src/gui/optionsdialog.h

```diff
@@ -210,4 +212,7 @@ private slots:
     AdvancedSettings *m_advancedSettings = nullptr;
 
     bool m_refreshingIpFilter = false;
+#ifndef DISABLE_WEBUI
+    QString m_currentAPIKey;
```

Comment:
Move it to the group at line 195.

Reply:
That group seems to be for function definitions, rather than member variables.